paddle/fluid/inference/api/analysis_config.cc

```diff
@@ -148,6 +153,28 @@ void AnalysisConfig::EnableMKLDNN() {
   Update();
 }
 
+void AnalysisConfig::EnableMkldnnQuantizer() {
+#ifdef PADDLE_WITH_MKLDNN
+  if (!mkldnn_quantizer_config_)
```

Comment:
if delete line158, then we don't need `CP_MEMBER(mkldnn_quantizer_config_)`?

Reply:
@luotao1 , copying the `shared_ptr` worked and it should be fine, even when it holds `nullptr`. I would rather suspect the `#ifdef PADDLE_WITH_MKLDNN` in the `paddle_analysis_config.h` is the culprit. I am investigating that now.

Reply:
@luotao1 , I confirm that removing `#ifdef PADDLE_WITH_MKLDNN` solved the problem. I took care of it in another way. Please take a look at this patch if it is OK for you: https://github.com/wojtuss/Paddle/commit/49c5a2b65c4e7be1457211352988e856203a69d8
I am verifying it more thoroughly.

Reply:
Got it. I verify on my local machine.